src/message/html/HtmlTagA.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
-import { Linking, Text } from 'react-native';
 
-import { getFullUrl } from '../../utils/url';
-import renderHtmlChildren from './renderHtmlChildren';
+import { isUrlInAppLink } from '../../utils/url';
+import HtmlRegularLinks from './HtmlRegularLinks';
+import HtmlInAppLinks from './HtmlInAppLinks';
 
-export default ({ href, auth, childrenNodes, cascadingStyle, onPress }) => (
-  <Text onPress={() => Linking.openURL(getFullUrl(href, auth.realm))}>
```

Comment:
Following the previous comments, this component will look very similar to the old version, but the `Linking.openURL` code will be somewhere in the parents.